### PR TITLE
Desktop version

### PR DESCRIPTION
**Pull request by Nedramrez**
Add style for the desktop version of  Headline Section
Add style for the desktop version of Projects Section
Add style for the desktop version of About me Section
Add style for the desktop version of  Contact me Section

### CODE REVIEW

File: styles.css [hunk 1]
```diff
@@ -377,3 +395,251 @@ form textarea {
...
+    margin-right: 10px;
+  }
+
+  .contact-form {
+    background-image: url(assets/images/contact-form-background-image.svg);
+    background-repeat: no-repeat;
+    background-position: center;
+    background-size: 100% 100%;
+  }
+
```
Comment (emmiiorji) [inline styles.css:629]: Your design looks so good and responsive too. Good job at that.

Kindly align the "Get in touch" button to the center as given by the design so we'll be as close to the design as possible.

Sticking to the design as much as possible (e.g. font, colors, images, text, margins) will be a good habit right now. You'll get more prepared for a typical workplace experience where you might be required to reproduce a given design as strictly as possible.

Your design:
image

Figma design:
image

**Review by emmiiorji — CHANGES_REQUESTED** [review]
Hi @Nedramrez, 
# Status: Changes Required ♻️

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

Almost there

## Good Highlights ✅

- Linter checks are passing
- Good commit messages
- PR is descriptive
- Adherence to GitHub flow
- Great design

## Required Changes ♻️

- Kindly remove unused files from your project. That will help not to clutter the project unnecessarily. You have committed an empty `pyasdf.py` file.
- _Check the comments under the review for more._

## Optional suggestions

- I suggest that you replace Microverse in the page's header with your name or logo since it's a personal portfolio.
image


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: styles.css (hunk 1 above)
Comment (Nedramrez) [reply styles.css:629]: Thank you!

**Review by emmiiorji — APPROVED** [review]
Hi @Nedramrez, 

# Status: Approved ✅

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

congratulations

## Good Highlights ✅

- Design is great and follows template closely
- Repo is professional
- Linter checks are passing
- Meaningful commit messages
- Implemented required and optional changes


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Nedramrez** [comment]
Thank you!

**Pull request merged**